types/google-apps-script-oauth2/README.md

```diff
@@ -0,0 +1,20 @@
+# Google Apps Script OAuth2
+
+Typescript definitions for the [Google Apps Script OAuth 2](https://github.com/googleworkspace/apps-script-oauth2) library
+
+## Installation
+
+```
+npm install --save-dev @types/google-apps-script-oauth2
+```
+
+Then, add it to your `tsconfig.json` under the `"types"` field:
```

Comment:
Based on https://www.typescriptlang.org/tsconfig#types, it would be included inherently since its a "public" package. So probably can skip this step.

Reply:
Hi @Blacksmoke16 only reason I'm here is to make VSCode happy, and step turned out to be the required one I needed :)

Reply:
Is this the only thing you have in that array? As if you do, then you'd have to also include it if everything isn't "public".

Reply:
No, I also needed to add `@types/google-apps-script`. Could be something related to my app's structure.

Reply:
Are all the other stuff in it `@types/` stuff? If so pretty sure you could just remove the whole array and it would work. Not a huge deal, i'd maybe add a sentence or so about how it may be optional and provide a link to https://www.typescriptlang.org/tsconfig#types for more information.

Reply:
thanks - yea I tried removing them all but for some reason it was required for me. I've updated with that link